### DISCUSSION TITLE
About running the toy model in document

### DISCUSSION THREAD

**Discussion by thisnameavailable** [dingo-gw/dingo#200] [Q&A] [answered]
I just installed dingo and want to try to use it following the toy model. When it comes to the training progress ,it raises ValueError ("Cannot update delta_f) shown in domains.py, line 134, update. I have no idea why it happens.

**Comment by stephengreen** [comment, marked as answer]
It looks like the example mistakenly uses different segment lengths for the noise ASD estimation (8s) and data segments (4s). To fix this, you should change the length of the ASD estimation segments below from 8s to 4s:

https://github.com/dingo-gw/dingo/blob/45c321dd43acb87874cf2c925551a15b9cc3eec2/examples/toy_npe_model/asd_dataset_settings.yaml#L6

We will also update the example with the correct setting.

**Reply by stephengreen** [reply]
This has now been fixed in by #204

**Reply by thisnameavailable** [reply]
thx a lot!!!
